action.yml

```diff
@@ -190,6 +190,12 @@ inputs:
       then one can specify 'gitHubEnterprise'.
     required: false
     default: 'gitHub'
+  pr-comment-github-token:
+    description: >
+      The Github Token for the repo that will be used to add a comment to the PR.
```

Comment:
```suggestion
      If specified, this action will post a PR comment containing the notebook output on run completion, using the specified GitHub token to post the comment. Note that no comment will be posted if this action is used in a workflow triggered from a source other than a pull request (e.g. a manually triggered workflow)
```

Reply:
Maybe something like this. Also, presumably my statement is correct that no comment gets posted if someone manually triggers the workflow?